include/grpcpp/ext/otel_plugin.h

```diff
@@ -106,6 +106,9 @@ class OpenTelemetryPluginBuilder {
   /// options can be added.
   OpenTelemetryPluginBuilder& AddPluginOption(
       std::unique_ptr<OpenTelemetryPluginOption> option);
+  // Records \a optional_label_key on all metrics that provide it.
+  OpenTelemetryPluginBuilder& AddOptionalLabel(
+      absl::string_view optional_label_key);
```

Comment:
This does not have any of the other changes mentioned in https://github.com/grpc/proposal/pull/421/files. Are they gonna come in a separate PR?

Reply:
Yeah, I think we will pair program that in a separate PR next :).